### PR TITLE
Migrates Cron e2e test to Go

### LINKED ISSUE #3217: Migrate e2e test for Cron to Go.
### Proposal

Relates to https://github.com/kedacore/keda/issues/2357, https://github.com/kedacore/keda/issues/2737.

### Use-Case

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Ritikaa96**
Signed-off-by: Ritikaa96 <ritika@india.nec.com>

Resolves #3217 

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)

### CODE REVIEW

**Review by JorTurFer — COMMENTED** [review]
LGTM, only small nit

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
 - **General:** Use Github's Checks API for e2e tests on PR. ([2567](https://github.com/kedacore/keda/issues/2567))
 - **General:** Migrate e2e test for Memory scaler to Go. ([3170](https://github.com/kedacore/keda/issues/3170))
 - **General:** Migrate e2e test for CPU scaler to Go. ([3169](https://github.com/kedacore/keda/issues/3169))
+- **General:** Migrate e2e test for Cron scaler to Go. ([3217](https://github.com/kedacore/keda/issues/3217))
```
[the remark this answers is not in the document]
Comment (Ritikaa96) [reply CHANGELOG.md]: > Also, the package name for the test.

I might have missed meanwhile correcting the previous change. what about the package name? i saw there are few more updates had been done to the main branch meanwhile. @v-shenoy

[the remark this answers is not in the document]
Comment (Ritikaa96) [reply CHANGELOG.md]: Aha! my bad. got it

**Review by JorTurFer — COMMENTED** [review]
LGTM,
Could you delete the old e2e test in ts? 🙏

**Review by JorTurFer — APPROVED** [review]
Thanks!

**Comment by v-shenoy** [comment]
I think some changes were missed here - https://github.com/kedacore/keda/blob/main/tests/run-arm-smoke-tests.sh

**Comment by JorTurFer** [comment]
🤦, you are right @v-shenoy 
could anyone create another PR fixing it?

**Pull request merged**